components/brave_rewards/browser/rewards_service_impl.cc

```diff
@@ -88,6 +88,8 @@ using net::registry_controlled_domains::INCLUDE_PRIVATE_REGISTRIES;
 using std::placeholders::_1;
 using std::placeholders::_2;
 
+#define RETRIES_COUNT_ON_NETWORK_CHANGE 1
```

Comment:
I'd probably prefer a `const static int` here to a define. I think we'd only use a define if we wanted to pass it in from gn

Reply:
same here, I would use `static const int kRetriesCountOnNetworkChange = 1` and move inside the namespace.

Reply:
Agreed.

Reply:
done